src/System/Logging/LogSink.hs

```diff
@@ -0,0 +1,45 @@
+module System.Logging.LogSink (
+  Sink(..)
+, LogLevel(..)
+, LogTarget(..)
+, setupLogging
+) where
+
+import           Control.Applicative
+import           System.Logging.Facade.Types
+import           System.Logging.Facade.Sink
+
+import qualified System.Logging.Facade as Log
+
+import           System.Logging.LogSink.Core
+import           System.Logging.LogSink.Format
+
+data LogTarget = StdErr | SysLog
+  deriving (Eq, Show)
+
+data Sink = Sink {
+  sinkLevel :: Maybe LogLevel
+, sinkFormat :: Maybe String
+, sinkTarget :: LogTarget
+} deriving (Eq, Show)
```

Comment:
@soenkehahn We now have that type in `System.Logging.LogSink.Core`.  We just don't have a type synonym for it.